package.json

```diff
@@ -8,16 +8,38 @@
     "start": "next start -p $PORT"
   },
   "dependencies": {
-    "@emotion/react": "11.4.1",
-    "@emotion/styled": "11.3.0",
-    "@mui/icons-material": "5.0.0-rc.0",
-    "@mui/material": "5.0.0-rc.0",
+    "@emotion/react": "11.7.0",
+    "@emotion/styled": "11.6.0",
+    "@gooddollar/goodprotocol": "1.0.0",
```

Comment:
needs to be updated to latest

Reply:
done i think:
goodprotocol "^1.0.0" ?
or should i specify the number? (1.0.2?)